### PR TITLE
Add README information for infile argument

### PR DESCRIPTION
**Pull request by Gabib32**
Provide information ahout the infile parser to the README for a
better user experience.

To close issue 76: https://github.com/clearlinux/autospec/issues/76

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ Synopsis
 
   usage: autospec.py [-h] [-g] [-n NAME] [-v VERSION]
                      [-a [ARCHIVES [ARCHIVES ...]]] [-l] [-b] [-c CONFIG]
-                     [-t TARGET] [-i] [-p] [--non_interactive] [-C]
+                     [-t TARGET] [-i] [-p] [--non_interactive] [-C] [--infile]
```
Comment (matthewrsj) [inline README.rst]: should be `[--infile INFILE]` to show it needs an argument.

File: README.rst [hunk 2]
```diff
@@ -87,6 +87,7 @@ optional arguments:
                         attempt to verify package
   -p, --prep-only       Only perform preparatory work on package
   --non_interactive     Disable interactive mode for package verification
+  --infile              Additional input file to contribute to specfile creation
```
Comment (matthewrsj) [inline README.rst]: Should be `--infile INFILE`

Should also mention that `INFILE` can be a directory containing bitbake files, a single file, or a url to a file

File: README.rst (hunk 1 above)
Comment (Gabib32) [reply README.rst]: Do you want to keep [url] as its own line? Is that intentional for formatting?

Comment (matthewrsj) [reply README.rst]: it's fine where it is.

**Comment by dorileo** [comment]
+1

**Comment by matthewrsj** [comment]
+1, thank you!

**Pull request merged**